darwin/Classes/SwiftAmplitudeFlutterPlugin.swift

```diff
@@ -82,6 +88,12 @@ import AmplitudeSwift
 
             result("serUserId called..")
 
+        case "getDeviceId":
+            let deviceId = amplitude?.getDeviceId()
+            amplitude?.logger?.debug(message: "Get device Id: \(String(describing: deviceId))")
```

Comment:
nit: the same as above
```suggestion
            amplitude?.logger?.debug(message: "Get deviceId: \(String(describing: deviceId))")
```